cmd/shadowsocks-httpget/httpget.go

```diff
@@ -14,6 +13,8 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	ss "github.com/bonafideyan/shadowsocks-go/shadowsocks"
```

Comment:
fix your dependency.

Reply:
your can add your branch as a reference for git

Reply:
Sorry, I don't know how to do it, could you please give me some steps in detail?